packages/api-client-core/src/AnyClient.ts

```diff
@@ -18,6 +19,7 @@ export interface AnyClient {
   mutate(graphQL: string, variables?: Record<string, any>): Promise<any>;
   transaction<T>(callback: (transaction: GadgetTransaction) => Promise<T>): Promise<T>;
   internal: InternalModelManagerNamespace;
+  $args: typeof $args;
```

Comment:
@infiton I think I remember you and @airhorns saying that you wanted to make the `$args` symbol available as part of `api` ?